public/components/Notifications/NotificationForm.tsx

```diff
@@ -15,7 +15,7 @@ import {
   EuiSpacer,
   EuiCompressedSwitch,
   EuiText,
-  EuiCompressedTextArea,
+  EuiCompressedTextArea, EuiCompressedCheckbox,
```

Comment:
NIt: Move `EuiCompressedCheckbox` to next line

Reply:
done